### PR TITLE
`readthedocs` infrastructure

### PR DESCRIPTION
**Pull request by EthanMarx**
Adds `readthedocs` infrastructure and starts moving existing documentation under the `docs` directory. An example build can be found here https://ethan-aframe.readthedocs.io/en/latest/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,95 +1,6 @@
 # Aframe
-Detecting compact binary mergers from gravitational wave strain data using neural networks, with an emphasis on
-- **Efficiency** - making effective use of accelerated hardware like GPUs in order to minimize time-to-solution
-- **Scale** - validating hypotheses on large volumes of data to obtain high-confidence estimates of model performance
-- **Flexibility** - modularizing functionality to expose various levels of abstraction and make implementing new ideas simple
-- **Physics first** - taking advantage of the rich priors available in GW physics to build robust models and evaluate them accoring to 
-meaningful metrics
-- **Multi-messenger astronomy** - making algorithmic decisions and optimizations that allow for extremely low-latency alerts 
+Detecting compact binary mergers from gravitational wave strain data using neural networks.
```
Comment (wbenoit26) [inline README.md:2]: Can we put a link to the documentation in the README? Or does the page not exist until this PR gets in?

File: docs/contributing.md [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
 
 To start contributing, create a [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of this repository. You can then clone your fork
 
 ```console
-git clone git@github.com:albert-einstein/aframev2.git
+git clone git@github.com:albert-einstein/aframe.git
 ```
 
-(replacing your GitHub username for `albert-einstein`). Then, add your the main (`ML4GW`) repository as a remote reference. A common practice is to rename this remote `upstream`.
+(replacing your GitHub username for `albert.einstein`). Then, add your the main (`ML4GW`) repository as a remote reference. A common practice is to rename this remote `upstream`.
```
Comment (wbenoit26) [inline docs/contributing.md:20]: Still referenced as `albert-einstein` in the example above

File: docs/projects/infer.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Infer
+Performing inference with `Aframe` models using NVIDIA's [Triton Inference Server](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/index.html)
+
+## Environment
+The infer project is currently not containerized, and only uses `poetry` for environment management.
```
Comment (wbenoit26) [inline docs/projects/infer.md:5]: Don't need to fix this now, but flagging it for correction later on

**Review by wbenoit26 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (EthanMarx) [reply README.md:2]: Yes. Will make a followup PR that does this (and addresses your below issues) once the build is successful

**Pull request merged**